backend/de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerBankAccountImportHelper.java

```diff
@@ -71,6 +69,9 @@ public I_C_BP_BankAccount importRecord(final I_I_BPartner importRecord)
 		if (bankAccount != null)
 		{
 			bankAccount.setIBAN(importRecord.getIBAN());
+			final String accountName = importRecord.getA_Name();
+			bankAccount.setA_Name(Check.isEmpty(accountName) ? importRecord.getSwiftCode() : accountName);
```

Comment:
Why not coalescing accountName, SwiftCode, then bpartnerName (swiftCode is also nullable, and bpartnerName is also used as value, see de.metas.payment.api.impl.QRPaymentStringDataProvider#createNewC_BP_BankAccount).

Reply:
In german this field is labeled as `Name auf Kreditkarte` (Name on CreditCard). This is why i didn't do coalescing. I don't know why Swift-Code was used here.
What do you think about using bppartner.name as fallback instead of Swift-Code?

Reply:
I see the bpartner name used in more recent code, so let's go with that then.